### ISSUE TITLE
SameSite support

### ISSUE THREAD

**Issue by xacx** [mozilla/node-client-sessions#143]
Are there plans for sameSite support for chrome 80 or is it alreay supported?

**Comment by xacx** [comment]
sameSite=none is not supported until package.json is updated. Temporary workaroud avaialble is
cd node_modules/client-sessions
npm install cookies@0.8.0

**Comment by jcheng5** [comment]
This is pretty important now :grimacing:

**Comment by drc-tjuszczyk** [comment]
is this in latest yet?

**Comment by drc-tjuszczyk** [comment]
@jcheng5 @xacx Can anyone still confirm that the cookies update is the only thing needed?

**Comment by jcheng5** [comment]
Yep--#142 works for me. (You do need to include `cookie: {sameSite: "none"}` in your `session()` options if you want that to be set, in case that wasn't obvious)

**Comment by drc-tjuszczyk** [comment]
the documentation just says it's "true" by default or something so no, it was no obvious lol. Thank you!

**Comment by jcheng5** [comment]
I mean, that's assuming that you do want your cookies to have SameSite=None. And if you do, you probably also need to use https://www.npmjs.com/package/should-send-same-site-none.

**Comment by xacx** [comment]
FYI, Chrome has rolled back the change https://blog.chromium.org/2020/04/temporarily-rolling-back-samesite.html

**Comment by vit100** [comment]
My buddy forked this repo and updated packages.json. If interested here it is:
https://github.com/nvanexan/node-client-sessions

and here is npm package: 
@nvanexan/node-client-sessions
